go.mod

```diff
@@ -22,24 +22,23 @@ require (
 	github.com/sergi/go-diff v1.1.0
 	github.com/sirupsen/logrus v1.8.1
 	github.com/spf13/afero v1.6.0
-	github.com/spf13/cobra v1.1.3
+	github.com/spf13/cobra v1.2.1
 	github.com/spf13/pflag v1.0.5
-	github.com/spf13/viper v1.7.0
+	github.com/spf13/viper v1.8.1
 	github.com/stretchr/testify v1.7.0
 	github.com/thoas/go-funk v0.8.0
-	golang.org/x/lint v0.0.0-20210508222113-6edffad5e616 // indirect
 	golang.org/x/mod v0.4.2
-	golang.org/x/tools v0.1.3
+	golang.org/x/tools v0.1.5
 	gomodules.xyz/jsonpatch/v3 v3.0.1
 	helm.sh/helm/v3 v3.6.2
-	k8s.io/api v0.21.2
-	k8s.io/apiextensions-apiserver v0.21.2
-	k8s.io/apimachinery v0.21.2
+	k8s.io/api v0.22.1
+	k8s.io/apiextensions-apiserver v0.22.1
+	k8s.io/apimachinery v0.22.1
 	k8s.io/cli-runtime v0.21.0
-	k8s.io/client-go v0.21.2
+	k8s.io/client-go v0.22.1
```

Comment:
we haven't bumped the dependencies to 1.22 yet, since [kb](https://github.com/kubernetes-sigs/kubebuilder/blob/master/go.mod) didn't bump it yet. Though this just bumps the deps in SDK binary and not in the plugin, do we wait till that is done?

Reply:
This is certainly different than what we've done in the past right? @camilamacedo86 is this really needed for this change?

Reply:
The API is bumped. So, when we bump the API that is automatic.
However, if I am not wrong our concern regards bump first kb is about the scaffold and not the tool. 
we are not changing the scaffold for the end-users here.

Reply:
But we can solved that :-) 

https://github.com/kubernetes-sigs/kubebuilder/pull/2339/files

Reply:
but we can update there as well: https://github.com/kubernetes-sigs/kubebuilder/pull/2339/

Reply:
As long as we're:
- not somehow using the deps of operator-sdk itself in the code that SDK scaffolds
- able to pin our test environments (kind, envtest, etc.) to use the versions we want, not the versions of SDK's deps

I don't see how this API bump would be a problem. Since no one should be able to import SDK code anymore, this should have no affect on anything except the operator-sdk binary itself.

Reply:
Thank you for your input and help 🥇 
I think the concern only comes because usually, we bumped KB first to also get the scaffolds updated as well.